config/supervisor/webhook/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+- manifests.yaml
```

Comment:
Don't we need a service like in the govmomi mode?

Reply:
Yes, we need it. As the same reason described above, it contains the service [here](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/config/webhook/service.yaml)

Reply:
I think we should be able to add supervisor/webhook/service.yaml after the steps described here: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2651#discussion_r1599891690

Reply:
Yes, added service.yaml for supervisor mode.